modules/space/rendering/renderablestars.cpp

```diff
@@ -674,11 +691,18 @@ bool RenderableStars::readSpeckFile() {
                 break;
             }
         }
+        minLumValue = values[4] < minLumValue ? values[4] : minLumValue;
+        maxLumValue = values[4] > maxLumValue ? values[4] : maxLumValue;
         if (!nullArray) {
             _fullData.insert(_fullData.end(), values.begin(), values.end());
         }
     } while (!file.eof());
 
+    // Normalize Luminosity:
+    for (size_t i = 0; i < _fullData.size(); i += _nValuesPerStar) {
+        _fullData[i + 4] = (_fullData[i + 4] - minLumValue) / (maxLumValue - minLumValue);
```

Comment:
Where does the 4 come from?

Reply:
From the definition of variables columns in the speck file.

Reply:
Just a quick comment there would be helpful for the future case when the speck file definition changes

Reply:
I changed the code and created a variable with a meaningful name equals to 4. :-)